editor/src/com/talosvfx/talos/editor/project2/GlobalDragAndDrop.java

```diff
@@ -71,10 +73,25 @@ public void dragStop (InputEvent event, float x, float y, int pointer, DragAndDr
 	}
 
 	public void fakeDragDrop (int screenX, int screenY, String[] absoluteFiles) {
+		if (absoluteFiles.length == 1) {
+			FileHandle handle = Gdx.files.absolute(absoluteFiles[0]);
+			if (handle.extension().equals(TalosProjectData.TALOS_PROJECT_EXTENSION)) {
+				// load the project
+				TalosProjectData data = TalosProjectData.loadFromFile(handle);
+				SharedResources.projectLoader.loadProject(data);
+				return;
+			}
+		}
+
 		ArrayDragAndDropPayload arrayDragAndDropPayload = new ArrayDragAndDropPayload();
 		Array<BaseDragAndDropPayload> items = arrayDragAndDropPayload.getItems();
 		for (String absoluteFileHandle : absoluteFiles) {
-			items.add(new FileHandleDragAndDropPayload(Gdx.files.absolute(absoluteFileHandle)));
+			FileHandle absolute = Gdx.files.absolute(absoluteFileHandle);
+			AssetFileDroppedEvent fileDropEvent = Notifications.obtainEvent(AssetFileDroppedEvent.class);
```

Comment:
Why are you sending an event? The thing that consumes the drag and drop through the drag and drop system is the only thing that cares about it

Reply:
Fixed!